manifests/install.pp

```diff
@@ -33,12 +33,18 @@
     require => Exec["pyenv::checkout ${user}"],
   }
 
-  exec { "pyenv::shrc ${user}":
-    command => "echo 'source ${pyenvrc}' >> ${shrc}",
-    user    => $user,
+  file { $shrc:
```

Comment:
`File[$shrc]`, might be defined elsewhere which could result in a redefinition of the resource. We can avoid this by checking for the existence of the resource first:

```puppet
if !defined(File[$shrc]) {
  file { $shrc:
    ...
  }
}
```